packages/url_launcher/lib/url_launcher.dart

```diff
@@ -39,7 +41,8 @@ Future<void> launch(
   String urlString, {
   bool forceSafariVC,
   bool forceWebView,
-  Brightness statusBarBrightness,
+  bool enableJavaScript,
+  Brightness statusBarBrightness = Brightness.light,
```

Comment:
I don't think this is merged correctly. `statusBarBrightness` needs to default to null.